File: ACSL/2-2018_2019C1/Answer.cpp
#include <iostream>
#include <string>
using namespace std;

int main(void)
{
    for (int test = 0; test < 5; test++)
    {
        string str;
        cin >> str;
        int ans = 0;
        for (int i = 0; i < str.length(); i++)
            if (test == 0)
                ans++;
            else if (test == 1)
                ans += str[i] - '0';
            else if (test == 2 && i % 2 == 0)
                ans += str[i] - '0';
            else if (test == 3)
                ans += str[i] - '0' == 4;
            else if (test == 4 && i == str.length() / 2 - 1 + str.length() % 2)
                ans = str[i] - '0';
        cout << ans << endl;
    }
    return 0;
}

File: ACSL/P2-2018_2019C1/Answer.py
for test in range(5):
    num_arr = input()
    ans = 0
    length = len(num_arr)
    for i in range(length):
        num = ord(num_arr[i]) - ord('0')
        if test == 0:
            ans += 1
        elif test == 1:
            ans += num
        elif (test == 2) and (i % 2 == 0):
            ans += num
        elif (test == 3) and (num == 4):
            ans += 1
        elif (test == 4) and (i == int(length / 2) - 1 + length % 2):
            ans = num
    print(ans)
